rio/src/RIO/Prelude/Logger.hs

```diff
@@ -315,7 +316,7 @@ logOptionsHandle handle' verbose = liftIO $ do
     , logVerboseFormat = return verbose
     , logTerminal = terminal
     , logUseTime = verbose
-    , logUseColor = verbose && terminal
+    , logUseColor = verbose && terminal && os /= "mingw32"
```

Comment:
I'd rather use the `WINDOWS` CPP flag, so that we use the same logic throughout the codebase for checking for Windows. Otherwise, looks great!